docs/log-in-with-techpass.md

```diff
@@ -28,7 +28,7 @@ This section guides you how to access services and products which use TechPass a
    
 5. If your TechPass log in ID's domain is ```techpass.gov.sg``` (for example: john_doe@techpass.gov.sg), enter the password of your TechPass account and click **Sign in**.
```

Comment:
if not then how?

Reply:
Not sure of this comment but as discussed, steps 4 and 5 start with a if clause for user to choose the required step.